solutions/USACO jan17-balanced-photo.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: usaco-693
+source: USACO Gold 2017 January
+title: Balanced Photo
+author: Andy Wang
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_bphoto_gold_jan17.html)
+
+The official solution uses [binary indexed tree](https://usaco.guide/gold/PURS/#binary-indexed-tree)
+
+An alternative solution uses order statistic tree.(C++ only)
+
+## Solution(C++ only)
```

Comment:
Put spaces between brackets and the preceding words

Reply:
put the period after the parentheses.